openshift-kube-apiserver/admission/storage/performantsecuritypolicy/admission.go

```diff
@@ -0,0 +1,121 @@
+package performantsecuritypolicy
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	openshiftfeatures "github.com/openshift/api/features"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/admission/initializer"
+	"k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/client-go/informers"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/component-base/featuregate"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+)
+
+const (
+	// Plugin name
+	PluginName               = "storage.openshift.io/PerformantSecurityPolicy"
+	fsGroupChangePolicyLabel = "storage.openshift.io/fsgroup-change-policy"
+	selinuxChangePolicyLabel = "storage.openshift.io/selinux-change-policy"
+)
+
+var _ = initializer.WantsExternalKubeInformerFactory(&performantSecurityPolicy{})
+var _ = admission.MutationInterface(&performantSecurityPolicy{})
+
+func Register(plugins *admission.Plugins) {
+	plugins.Register(PluginName,
+		func(config io.Reader) (admission.Interface, error) {
+			return &performantSecurityPolicy{
+				Handler: admission.NewHandler(admission.Create),
+			}, nil
+		})
+}
+
+// csiInlineVolSec validates whether the namespace has permission to use a given
+// CSI driver as an inline volume.
```

Comment:
This looks like copy-paste leftover, I hope we don't combine performantSecurityPolicy with the inline CSI volume security.